osquery/tables/system/darwin/apps.mm

```diff
@@ -55,6 +55,7 @@
 const std::vector<std::string> kSystemSearchPaths = {
     "/Applications",
     "/System/Library/Core Services/Applications",
+    "/Library/Apple/System/Library/CoreServices",
```

Comment:
This part didn't help. Still testing

Reply:
Ah. I think I see what's happening...

the apps table tries to use the `com.apple.LaunchServices` api to enumerate apps, and only uses these paths if that fails.

This seems reasonable to me, though it does mean we miss apps apple has not registered there. I think this is probably okay for now